src/handlebars/release_notes.handlebars

```diff
@@ -26,10 +26,10 @@
 
     <div class="anchor">
       <a href="#jdk8u242" class="anchor"><img src="dist/assets/anchor.png" alt="anchor icon"></a>
-      <h2 class="bold"><a id="jdk8u242">OpenJDK 8u242 (1.8.0_242-b08)</a></h2>
+      <h2 class="bold"><button type="button" onclick="showHideReleaseNotes('jdk8u242_notes');">-/+</button> <a id="jdk8u242">OpenJDK 8u242 (1.8.0_242-b08)</a></h2>
       <h3 style="text-align: center;">(<a href="https://github.com/AdoptOpenJDK/openjdk8-upstream-binaries/releases/download/jdk8u242-b08/OpenJDK8U-sources_8u242b08.tar.gz">Upstream Source Tarball OpenJDK 8u242-b08</a>)</h3>
     </div>
-    <div class="margin-bottom">
+    <div class="margin-bottom hide" id="jdk8u242_notes">
```

Comment:
The [hide](https://github.com/AdoptOpenJDK/openjdk-website/blob/91b3febe260a098afd15431f75eed4afcdbe989a/src/scss/styles-1-large-main.scss#L69) class sets `display: none !important`, which overrides the style.display value set by the script:
> ![image](https://user-images.githubusercontent.com/5124113/80027401-52b2a400-84a9-11ea-893a-73965f23a808.png)


Replace the `hide` class with an inline style for a quick fix:
```suggestion
    <div class="margin-bottom" id="jdk8u242_notes" style="display: none;">
```